iceoryx_meta/CMakeLists.txt

```diff
@@ -67,6 +67,8 @@ if(EXAMPLES)
     add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../iceoryx_examples/ice_multi_publisher ${CMAKE_BINARY_DIR}/iceoryx_examples/ice_multi_publisher)
 endif()
 
+include(${CMAKE_CURRENT_SOURCE_DIR}/../cmake/package/package.cmake)
```

Comment:
Just out of curiosity: what is this for?

Reply:
This is the cmake file for the debian package which we include here